projects/schematics/src/migrations/4_0/constructor-deprecations/data/configurator-place-order-hook.effect.migration.ts

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
I think for the effect we don't need the migration, as not publicly exposed. @Platonn Correct me if I am wrong

Reply:
Correct, If it's not part of the Public API, we don't need the migration script.  I'm removing it.